packages/gatsby-telemetry/src/telemetry.js

```diff
@@ -60,7 +74,7 @@ module.exports = class AnalyticsTracker {
     }
     return undefined
   }
-  captureEvent(type = ``, tags = {}) {
+  captureEvent(type = ``, tags = {}, opts = { debounce: false }) {
```

Comment:
Want to set debounce: true by default?

Reply:
I'm slightly worried about the possible side-effects so hence I'd leave it as it was (i.e. no debouncing by default)